projects/client/RabbitMQ.Client/src/client/api/IConnectionFactory.cs

```diff
@@ -129,13 +129,6 @@ public interface IConnectionFactory
         IConnection CreateConnection(IList<string> hostnames, String clientProvidedName);
 
         /// <summary>
-        /// Advanced option.
-        ///
-        /// What task scheduler should consumer dispatcher use.
-        /// </summary>
-        TaskScheduler TaskScheduler { get; set; }
```

Comment:
Yes but it will still compile and function against current code, albeit with a warning. This will give users an opportunity to review and update their code for when the property is properly removed later.